packages/app/app/components/RepoSearch.vue

```diff
@@ -1,12 +1,55 @@
 <script lang="ts" setup>
+import type { RepoNode } from "../../server/utils/types";
 const search = useSessionStorage("search", "");
+const searchResults = ref<
+  Array<{
+    id: string;
+    name: string;
+    owner: { login: string; avatarUrl: string };
+    stars?: number;
+  }>
+>([]);
+const isLoading = ref(false);
 
+let activeController: AbortController | null = null;
```

Comment:
now i'm thinking this might need to be a ref too!

Reply:
doesn't vue have a primitive for non-reactive values?

Reply:
if it has, then let's use it! i prefer to use the native primitives of a framework.